redi-connect-front/src/components/organisms/Checklist.scss

```diff
@@ -0,0 +1,58 @@
+@import "~bulma/sass/utilities/_all";
+@import "../../styles/_variables.scss";
+
+.checklist {
+  &__headline {
+    padding-bottom: 4.5rem;
```

Comment:
Distance between element should be defined trough margin.

Reply:
adjusted